.eslintplugin/index.js

```diff
@@ -9,4 +9,6 @@ exports.rules = {
   'ban-types-eventually': require('@typescript-eslint/eslint-plugin').rules[
     'ban-types'
   ],
+  'prefer-rest-params-eventually': require('eslint/lib/rules/prefer-rest-params'),
```

Comment:
So these exist off of `require('eslint')` instead of the deep import?

Reply:
Do you mean "do"?

If so then, no they don't - because `eslint` is the "core", it doesn't export its rules in the same way plugins do, so you can't do `require('eslint').rules['<rule>']`.

In fact, it doesn't export its rules at all:

```
❯ node
Welcome to Node.js v14.4.0.
Type ".help" for more information.
> require('eslint')
{
  Linter: [Function: Linter],
  CLIEngine: [Function: CLIEngine] {
    version: '7.8.1',
    getFormatter: [Function: getFormatter]
  },
  ESLint: [Function: ESLint],
  RuleTester: [Function: RuleTester] {
    [Symbol(it)]: null,
    [Symbol(describe)]: null
  },
  SourceCode: [Function: SourceCode]
}
```

Reply:
👍